css/codemirror.css

```diff
@@ -333,3 +333,36 @@ div.CodeMirror-dragcursors {
 
 /* Help users use markselection to safely style text background */
 span.CodeMirror-selectedtext { background: none; }
```

Comment:
Couldn't find a better spot to add these additions, which come straight from CodeMirror